Agents/CEAAgent/README.md

```diff
@@ -10,70 +10,47 @@ The agent currently queries for building geometry, surrounding buildings geometr
 ### Maven
 
 The docker image uses TheWorldAvatar maven repository (https://maven.pkg.github.com/cambridge-cares/TheWorldAvatar/).
-You'll need to provide your credentials (github username/personal access token) in single-word text files located:
+You will need to provide your credentials (github username/personal access token) in single-word text files located:
 ```
 ./credentials/
     repo_username.txt
     repo_password.txt
 ```
+### Stack Set Up
+The agent is designed to run in the stack. To start the stack, spin up the [Stack Manager](https://github.com/cambridge-cares/TheWorldAvatar/blob/main/Deploy/stacks/dynamic/stack-manager).
 
-### PostgreSQL
+### Access Agent Set Up
+The agent is designed to use the [JPS Access Agent](https://github.com/cambridge-cares/TheWorldAvatar/tree/1579-dev-stackerise-cea-agent/JPS_ACCESS_AGENT). Spin the access agent up as part of the same stack as spun up by the Stack Manager.
```

Comment:
This link will become invalid as soon as this branch is merged. You may want to replace the URL with `../AccessAgent`. That is not currently a valid path either, but will be soon (once `dev-main-readme` is merged and I've moved the access agent to its new location...).